content/en/blog/2021-08-15-seccomp-default.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: blog
+title: "Hardening Kubernetes security by enabling seccomp by default"
+date: 2021-08-15
+slug: seccomp-default
+---
+
+**Author:** Sascha Grunert, Red Hat
+
+_Editor's note: Sascha is member of [SIG Release][sigr] and is working on many
+other different container related topics. Feel free to reach him out on Twitter
+[@saschagrunert][twitter] or via the [Kubernetes slack @sascha][slack]._
+
+[sigr]: https://github.com/kubernetes/sig-release
+[twitter]: https://twitter.com/saschagrunert
+[slack]: https://kubernetes.slack.com/team/U53SUDBD4
+
+---
+
+Applying seccomp profiles to containerized workloads is one of the key tasks
+when it comes to enhancing the security of the application deployment.
+Developers, site reliability engineers and infrastructure administrators have to
+work hand in hand to create, distribute and maintain the profiles over the
+applications life-cycle.
+
+Kubernetes introduced dedicated [API fields][api] in the `SecurityContext` with
```

Comment:
A reader might infer that they can `kubectl get SecurityContext` in their cluster. We should explain that `securityContext` is a field you find in Pods and in pod templates.

Reply:
Added more explanation to the paragraph :)